common/src/main/java/dev/ftb/mods/ftblibrary/FTBLibraryClient.java

```diff
@@ -114,6 +115,7 @@ private void clientTick(Minecraft client) {
 	}
 
 	public static boolean areButtonsVisible(@Nullable Screen gui) {
+		if (Platform.isForge() && Platform.isModLoaded("roughlyenoughitems")) return false;
```

Comment:
TODO: remove Platform.isForge() call in 1.17+, this is only here because Fabric 1.16 still uses REI 5

Reply:
Also maybe we could just set showButtons to 0 at init if REI is loaded? Same effect as having it here but it removes the duplicate platform check.